fundamental-concepts/data-structures-stdlib/6_stdlib.md

```diff
@@ -0,0 +1,68 @@
+# Importing modules 2/2: Importing the standard library
+
+The same way you import your own module, you can import modules from the [standard
+library](https://docs.daml.com/daml/stdlib/index.html). Let's import the `DA.Time` module and add a
+birthday date to our `Person` data type in `daml/AddressBook.daml`{{open}}:
+
+<pre class="file" data-target="clipboard">
+import DA.Date
+</pre>
+
+Now we can extend the `Person` record in `daml/AddressBook`{{open}}
+
+<pre class="file" data-target="clipboard">
+data Person = Person {
+  name: Text,
+  address: Address,
+  age: Int,
+  birthday: Date
+}
+</pre>
+
+And add the missing field to `testPerson`:
+
+<pre class=file data-target="clipboard">
+testPerson: Person
+testPerson = Person {name = "Alice", address = Address {street = "RabbitStreet", city = "TheBigUnknown", country = "Wonderland"}, age = 7, birthday = date 30 Oct 2013}
+</pre>
+
+The functions `date` and the constructor `Oct` are provided by the `DA.Date` module of the standard
+library. In the next steps we look at some of the most commonly used data types and functions
+provided by the standard library.
+
+The same way you import the standard library you can import modules of your own packages by adding
+an import statement
+
+<pre>
+import AddressBook
+</pre>
+
+to your source, after you have compiled the `organizer` package with 
+
+```
+cd organizer
+daml build
+```{{execute T1}}
+
+and made it available by adding it to your `daml.yaml` file under the `dependencies` stanza:
+
+<pre>
+sdk-version: 1.2.0
+name: some-other-package
+version: 0.1.0
+source: daml
+parties:
+- Alice
+- Bob
+- Charlie
+- drsk
+dependencies:
+- daml-prim
+- daml-stdlib
+- daml-trigger
+- ../organizer/organizer-0.1.0.dar
+sandbox-options:
+- --wall-clock-time
+- --ledgerid=organzier-sandbox
+start-navigator: false
+</pre>
```

Comment:
```suggestion
The same way you import the standard library you can import modules of your own packages. You can do this in three easy steps:

1. First compile the package with the module you want to import. In our case this would be the `organizer` package 

**<!-- add tripple backticks -->** 
cd organizer
daml build
{{execute T1}}

2. Next add the compiled package to your `daml.yaml` file under the `dependencies` stanza:

<pre>
sdk-version: 1.2.0
name: some-other-package
version: 0.1.0
source: daml
parties:
- Alice
- Bob
- Charlie
- drsk
dependencies:
- daml-prim
- daml-stdlib
- daml-trigger
- ../organizer/organizer-0.1.0.dar
sandbox-options:
- --wall-clock-time
- --ledgerid=organzier-sandbox
start-navigator: false
</pre>

3. The very last step is to add the `import` statement for the module you want to work with to your `.daml` file. In our case this would be 

<pre>
import AddressBook
</pre>
```

I think this is the correct order of how you would do it, right? So you would:
1. Compile the package
2. Add it to dependencies
3. Import the module

Do you think we should give more info on why these exact steps need to be done? Or it would be too much?

FYI had to remove the tripple backticks from the terminal command as GH was breaking the suggested change.

Reply:
I added this with small changes.

Reply:
Hey there 😄 Where did you add the changes? Can't see them.